README.md

```diff
@@ -303,6 +303,67 @@ retriable_with_context(:api) do
 end
 ```
 
+## In Tests
```

Comment:
this all looks great but maybe you can retitle this?  when i first red this i assumed it was about running tests for the gem... so maybe something like "Short Circuiting Retriable While Testing Your App"

i might also add a note about what exactly the problem is (namely, that it often takes a long time to retry blocks that fail)

Reply:
@apurvis 425f2d9. Let me know if that is clearer. I can squash commits if this looks good.